raft/src/main/java/org/apache/kafka/raft/LeaderState.java

```diff
@@ -76,9 +85,37 @@ protected LeaderState(
             boolean hasAcknowledgedLeader = voterId == localId;
             this.voterStates.put(voterId, new ReplicaState(voterId, hasAcknowledgedLeader));
         }
+        this.majority = voters.size() / 2;
         this.grantingVoters = Collections.unmodifiableSet(new HashSet<>(grantingVoters));
         this.log = logContext.logger(LeaderState.class);
         this.accumulator = Objects.requireNonNull(accumulator, "accumulator must be non-null");
+        this.fetchTimeoutMs = fetchTimeoutMs;
+        this.fetchTimer = time.timer(fetchTimeoutMs);
+    }
+
+    public boolean hasMajorityFollowerFetchTimeoutExpired(long currentTimeMs) {
```

Comment:
nit: these are pretty lengthy and arguably unintuitive method names, could we add a short comment on what each method does?

Reply:
Changed to `hasMajorityFollowerFetchExpired`. Let me know if you have any better suggestion. Thanks.

Reply:
I was more suggesting that this method might benefit from a comment which describes behavior. I guess the info log explains it well enough

Reply:
Fair enough. Added comments on the methods. Thanks.